src/python/WMCore/WMSpec/WMWorkloadTools.py

```diff
@@ -246,17 +245,8 @@ def validatePhEDExSubscription(arguments):
     for site in arguments.get("AutoApproveSubscriptionSites", []):
```

Comment:
I think we should take advantage and also drop this parameter `AutoApproveSubscriptionSites`.

Reply:
Done with https://github.com/dmwm/WMCore/pull/11181/commits/f12b5f5c3e9ef2e305b1590c32728cb1a56be26c and https://github.com/dmwm/WMCore/pull/11181/commits/0c5b7562e64ae0de63470d9df5368e2c2d464727